modules/oe_theme_helper/templates/retina-image-scale-summary.html.twig

```diff
@@ -0,0 +1,37 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a summary of an image scale effect.
+ *
+ * Available variables:
+ * - data: The current configuration for this resize effect, including:
+ *   - width: The width of the resized image.
+ *   - height: The height of the resized image.
+ *   - upscale: If images larger than their original size can scale.
+ * - effect: The effect information, including:
+ *   - id: The effect identifier.
+ *   - label: The effect name.
+ *   - description: The effect description.
+ *
+ * @ingroup themeable
+ */
+#}
+{% if data.width and data.height -%}
```

Comment:
Why is this needed? You copy pasted this entire thing and made no changes whatsoever..

Reply:
The twig template is needed in because the image style preview requires it. Since the changes we made to the base scale image style are a very edge case I didn't think we would need to change anything on the summary template either.

Reply:
Then why not use the original?!

Reply:
Fair enough, I'll re-use the original template.